index.js

```diff
@@ -99,8 +99,7 @@ module.exports = class ProtomuxRPC extends EventEmitter {
         value = await responder.handler(value)
       } catch (err) {
         safetyCatch(err)
-
-        error = err.message
+        error = err
```

Comment:
```suggestion

        error = err
```

Just for consistency.